gordo_components/__init__.py

```diff
@@ -51,12 +51,3 @@ def _parse_version(version: str) -> Tuple[int, ...]:
 except Exception:
     warnings.warn(f"Failed to fix absl logging bug {traceback.format_exc()}")
     pass
-
-
-# Set log level, defaulting to INFO
-log_level = os.getenv("LOG_LEVEL", "INFO").upper()
-
-logging.basicConfig(
-    level=getattr(logging, log_level),
-    format="[%(asctime)s] %(levelname)s [%(name)s.%(funcName)s:%(lineno)d] %(message)s",
-)
```

Comment:
Do you want this change in this PR?

Reply:
No, you are correcto!

Reply:
but, I will keep it here until #774 is merged because it's a right pain to debug when tests fail. After that's in, the diff will be gone.